File: Python/0061.py
team1, team2 = 0, 0

for _ in range(4):
    points1, points2 = map(int, input().split())
    team1 += points1
    team2 += points2

if team1 > team2:
    print('1')
elif team1 < team2:
    print('2')
else:
    print('DRAW')

File: C++/0061.cpp
#include <iostream>
using namespace std;

int main()
{
    int firstTeam = 0, secondTeam = 0;
    int points1, points2;

    for (int i = 0; i < 4; i++)
    {
        cin >> points1 >> points2;
        firstTeam += points1;
        secondTeam += points2;
    }

    if (firstTeam > secondTeam)
        cout << "1" << endl;
    else if (firstTeam < secondTeam)
        cout << "2" << endl;
    else
        cout << "DRAW" << endl;

    return 0;
}
